networkx/algorithms/bipartite/tests/test_extendability.py

```diff
@@ -0,0 +1,415 @@
+import networkx as nx
+from networkx.algorithms.bipartite import extendability
```

Comment:
You can leave this out and for each test use `nx.bipartitie.find_extendability`  which will be the way we suggest this function be called.